install/templates/prometheus.tmpl

```diff
@@ -28,16 +28,7 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 networks:
-  # Bridge so node-exporter can get the real NIC details
-  # See https://stackoverflow.com/a/66689508 for more info
```

Comment:
The stackoverflow solution refered in the comment is obselete by now. Adding "host.docker.internal:host-gateway" as extra host is enough alone. The stackoverflow answer was updated to reflect that.